drinfomon.lic

```diff
@@ -386,11 +433,6 @@ class DRSkill
   # When it was defined as a class method then the initialize method
   # complained that this method didn't yet exist.
   def lookup_skillset(skill)
-    # To mitigate slowing down drinfomon loading up, the skills
-    # data is lazily loaded instead of loaded as part of the class variable
-    # initialization. The delay caused other scripts to complain that `DRRoom`
-    # had not been initialized yet.
-    @@skills_data ||= get_data('skills')
```

Comment:
I'm not seeing the "undefined" issue anymore, so have removed this lazy load. The bigger motivator for refactoring this to be done at the class level instead in this method is I need the `@@skills_data` is needed in both the instance method `lookup_skillset(..)` and the class method `self.lookup_alias(..)` and I kept getting method not yet defined errors when trying to have the instance method call the class method.

So instead of having `@@skills_data ||= get_data('skills')` repeated in both methods, I just put it back at the top of the class declaration.